DNN Platform/Library/Entities/Portals/PortalController.cs

```diff
@@ -2923,12 +2923,6 @@ private void ParsePortalSettings(XmlNode nodeSettings, int portalId)
                 UpdatePortalSetting(portalId, "DefaultAdminContainer", XmlUtils.GetNodeValue(nodeSettings, "containersrcadmin", string.Empty), true, currentCulture);
             }
 
-            // Enable Skin Widgets Setting
-            if (!string.IsNullOrEmpty(XmlUtils.GetNodeValue(nodeSettings, "enableskinwidgets", string.Empty)))

```

Comment:
Technically this change, along with the change in the PortalSettingsController.cs file introduce breaking changes.  In the past we have NOT removed functionality until the actual property is removed.  If someone, for some reason, was using this setting it would now be broken.  I do not believe that these changes should be included now as it doesn't match the spirit of our deprecation process.